### DISCUSSION TITLE
Even changing the instance I can't play some videos.

### DISCUSSION THREAD

**Discussion by MarlusArthur** [libre-tube/LibreTube#3174] [General]
hello! I live in Brazil and I have difficulty playing some videos on Libretube, the app asks me to change the instance, but when I do that the video still doesn't work. to be more exact I've tried with all available instances and nothing. these videos that don't work are from some youtubers from Brazil. I'll leave an example link.

***the video worked perfectly in the YouTube app

https://youtu.be/88jWB1TBw0s

**Comment by Bnyro** [comment]
See https://github.com/libre-tube/LibreTube/issues/3051

**Reply by MarlusArthur** [reply]
hello! even activating the HLS the video did not run. I was talking to a Telegram user and he asked me for the video link, this user lives in another country and he couldn't play the video (he tried in the YouTube app too) because this video seems to have a region lock and only can be accessed by Brazilians. however for me in Libretube it doesn't work and in the YouTube app the video works perfectly.
